clone_repos.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+MIRROR="https://github.com"
+
+function install_rpm () {
+    yum install $1
+}
+
+function install_git () {
+    git clone $MIRROR/$2 -b $3 $1
+    cd $1
+    python setup.py install
+    cd ..
+}
+
+
+# Note: versions are train.  We should try stein...
+
+install_rpm python36-pytz
+
+# epel? :(
+install_rpm python36-pbr
+# pbr!=2.1.0,>=2.0.0 # Apache-2.0
+
+# epel? :(
+install_rpm python36-sqlalchemy
+# SQLAlchemy!=1.1.5,!=1.1.6,!=1.1.7,!=1.1.8,>=1.0.10 # MIT
+
+# epel? :(
+install_rpm alembic
+install_rpm alembic-libs
+# TODO https://files.pythonhosted.org/packages/7b/8b/0c98c378d93165d9809193f274c3c6e2151120d955b752419c7d43e4d857/alembic-1.0.11.tar.gz
+# alembic>=0.8.10 # MIT
+
+install_git automaton openstack/automaton 1.16.0
```

Comment:
can we have a separate file with just
```
name repo version [mirror]
```
like
```
automaton openstack/automaton 1.16.0
```
bonus for auto-detection that `openstack` and `x` should be fetched from opendev (actually, `x` stuff doesn't have a github mirror).

Reply:
That is likely overcomplicating :\  Minimal logic may be better in the short term.